addons/mail/static/src/js/client_action.js

```diff
@@ -654,6 +766,12 @@ var ChatAction = Widget.extend(ControlPanelMixin, {
                 .find('.o_mail_chat_button_unstar_all')
                 .toggleClass('disabled', disabled);
         }
+        if (this.channel.id === "channel_moderation" || Boolean(this.channel.moderation && this.channel.is_moderator)) {
+            this.thread.trigger('render_select_button');
```

Comment:
why do you do that? looks like you want to call the _updateSelectAllButton method, so why don't you directly call it?